ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -191,8 +191,12 @@ - (dispatch_queue_t)methodQueue {
     [Instabug clearFileAttachments];
 }
 
-RCT_EXPORT_METHOD(identifyUser:(NSString *)email name:(NSString *)name) {
-    [Instabug identifyUserWithEmail:email name:name];
+RCT_EXPORT_METHOD(identifyUser:(NSString *)email name:(NSString *)name id:(nullable NSString *)id) {
+    [Instabug identifyUserWithID:id email:email name:name];
+}
+
+RCT_EXPORT_METHOD(identifyUserWithID:(NSString *)userID email:(NSString *)email name:(NSString *)name) {
+    [Instabug identifyUserWithID:userID email:email name:name];
```

Comment:
Is there a need for having both `identifyUserWithID` and `identifyUser`? As per our last discussion, I think we agreed on having the two scenarios handled in one method instead, please correct me if I'm wrong.

Also, did we verify that not passing the ID or passing it as `null`/`nil` would work fine?

Reply:
> Is there a need for having both identifyUserWithID and identifyUser? As per our last discussion, I think we agreed on having the two scenarios handled in one method instead, please correct me if I'm wrong.
Yes you're right, I think I forgot to remove it from here.

> Also, did we verify that not passing the ID or passing it as null/nil would work fine?
Yes It's backwards compatible and works just fine.